### PR TITLE
docs:  improve on Site URIs setting

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
See https://forum.codeigniter.com/showthread.php?tid=88363

- add about `$indexPage`

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

**Comment by kenjis** [comment]
Can someone review?

File: user_guide_src/source/installation/running.rst [hunk 1]
```diff
@@ -25,31 +25,53 @@ section of the User Guide to begin learning how to build dynamic PHP application
...
-   (Always use a trailing slash on your base URL!)
+Configure for Your Site URIs
+============================
+
+Open the **app/Config/App.php** file with a text editor.
+
+#. $baseURL
+    Set your base URL to ``$baseURL``. If you need more flexibility, the baseURL may
+    be set within the :ref:`.env <dotenv-file>` file as ``app.baseURL = 'http://example.com/'``.
+    (Always use a trailing slash on your base URL!)
```
Comment (datamweb) [inline user_guide_src/source/installation/running.rst:36]: I saw some people having problems in the forum. I think it is prominently bold.

Comment (kenjis) [reply user_guide_src/source/installation/running.rst:36]: You mean to make "Always use a trailing slash on your base URL!" bold?

Comment (datamweb) [reply user_guide_src/source/installation/running.rst:36]: Sorry. Yes.

Comment (kenjis) [reply user_guide_src/source/installation/running.rst]: Done!

**Review by datamweb — APPROVED** [review]
Thank you!

**Comment by kenjis** [comment]
@datamweb Thank you for the review!

**Comment by neznaika0** [comment]
@kenjis is there some important reason not to remove the index? Most people ask a question and no one wants to have an index in the URL. Should it be disabled by default?

**Comment by kenjis** [comment]
It came from CI3. Probably, the URI with `index.php` always works, but the URI without it needs mod_rewrite on Apache.

**Pull request merged**